AppController/djinn.rb

```diff
@@ -3553,15 +3553,7 @@ def assign_datastore_processes
     # machine.
     db_nodes.each { |node|
       assignments = {}
-      begin
-        cpu_count = HermesClient.get_cpu_count(node.private_ip, @@secret)
-        server_count = cpu_count * DatastoreServer::MULTIPLIER
-      rescue FailedNodeException
-        server_count = DatastoreServer::DEFAULT_NUM_SERVERS
-      end
-
-      assignments['datastore'] = {'count' => server_count,
-                                  'verbose' => verbose}
+      assignments['datastore'] = {'verbose' => verbose}
```

Comment:
At some point we were considering autoscaling for datastore processes. In the future, there may be some process that always sets count and adjusts it based on demand and availability. However, we don't have any of that implemented yet, and it may be some time before we are able to.

In the meantime, I'm good with this fix.